### ISSUE TITLE
Decoding password of proxy settings

### ISSUE THREAD

**Issue by hoppmann** [igvteam/igv#1011]
Hi,

I just noticed, that if you enter a password for the proxy settings the password will be Base64 encoded. But the encoded password is not decoded, if you check your proxy settings afterwards. In the password textfield there is always the encoded password written.

**Comment by jrobinso** [comment]
That's possibly by design, I would have to check, its still not secure but maybe a little better than plain text.

I don't have a password protected proxy to test with, so I'm hesitant to even touch this.   Is there a problem, other than you can't see the password in the editor?  Is the proxy working with it?

**Comment by jrobinso** [comment]
Actually now that you've raised this issue, I don't think we should be storing passwords at all in any form.  Does your system work if you remove these settings?  You should get a login popup triggered from the proxy challenge.   To totally remove the settings it might be necessary to shut down IGV, then delete the following properties from "pref.properties"

PROXY.USERNAME
PROXY.PW

Please respond when you can test, I am considering removing these properties.

**Comment by hoppmann** [comment]
Hi,
I've just testet it. I shut down IGV, deleted the prefs.property and started it again. But I didn't get a popup for entering the proxy settings. I know that I saw this popup once, when I initially started IGV for the first time ever. But since then it dosn't pop up anymore.

**Comment by jrobinso** [comment]
@hoppmann OK, follow on question,  is there an actual problem (IGV doesn't work), or is it just that you can't see the password you entered in plain text in the preferences window?

I'm looking into options for setting up a proxy for testing.

**Comment by jrobinso** [comment]
The editor problem should be fixed, and will be in the next release.   From code inspection I think this was just an editor display issue.

**Comment by hoppmann** [comment]
Hi,
It was mainly, the Issue with the editor. I came across some Issues running IGV behind a proxy since I don't get this pop-up window for entering Proxy-Settings, and wanted to post it here. But then I could solve it after some more issues, via the permanent proxy settings. 
Since I stumbled over the display issue, I thought I send a short notice. Actually, I would prefere if the settings within the file would be kept. I'm using IGV via my own Java tool. And with the proxy settings in the preference-file I can set these setting in my program instead having other users have to manage IGV settings.

Thanks,
Anselm
